.travis.yml

```diff
@@ -18,7 +18,7 @@ python:
   - "2.7"
   - "3.5"
   - "3.6"
-  - "3.7-dev"
+#  - "3.7-dev"
```

Comment:
Instead of removing it, what about putting this under the matrix section below (same depth as `include`):
```yml
matrix:
  allow_failures:
    - python: "3.7-dev"
```